resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.spec.erb

```diff
@@ -132,7 +159,6 @@ Requires: puppet-agent
 Obsoletes: <%= proj_name -%>-terminus < <%= info[:version].gsub('-', '.') %>
 Provides:  <%= proj_name -%>-terminus >= <%= info[:version].gsub('-', '.') %>
 <% end -%>
-
```

Comment:
I think erb will actually trim this space since the tag above this is `-%>`, lemme double check